### PR TITLE
Switch to use our fork of jira-client

### PR DESCRIPTION
**Pull request by mtscout6**

### CODE REVIEW

File: src/jira-connection.js [hunk 1]
```diff
@@ -1,40 +1,29 @@
...
+class OurJiraApi extends JiraApi {
+  constructor(config) {
+    super(config);
+  }
 
-Object.keys(JiraApi.prototype).forEach(key => {
-  const currentProperty = JiraApi.prototype[key];
+  findIssue(issueNumber) {
+    if (!this._findIssueCache) {
+      this._findIssueCache = _.memoize(super.findIssue);
```
Comment (KanoYugoro) [inline src/jira-connection.js]: Not saying that this is correct, but couldn't one write this as:

```javascript
this._findIssueCache = this._findIssueCache || _.memoize(super.findIssue);
```

**Comment by KanoYugoro** [comment]
LGTM

File: src/jira-connection.js (hunk 1 above)
Comment (mtscout6) [inline src/jira-connection.js]: That is a valid way to do it, either way will work.

**Pull request merged**